objects.lisp

```diff
@@ -367,3 +451,75 @@
                      (last-status last-status)
                      (unread unread)) conversation
       (format stream "~a unread? ~a ~a ~a" id unread accounts last-status))))
+
+(define-entity featured-tag
+  (id)
+  (name)
+  (statuses-count :field "statuses_count" :translate-with #'parse-integer)
+  (last-status-at :field "last_status_at" :translate-with #'convert-timestamp))
+
+(defmethod print-object ((featured-tag featured-tag) stream)
+  (print-unreadable-object (featured-tag stream :type T)
+    (with-accessors ((id id)
+                     (name name)
+                     (statuses-count statuses-count)
+                     (last-status-at last-status-at)) featured-tag
+      (format stream
+              "~a name ~a  count ~a last status at ~a"
+              id name statuses-count last-status-at))))
+
+(define-entity filter
+  (id)
+  (phrase)
+  (filter-context)
```

Comment:
As mentioned above in the comment, please translate this using either `to-keyword`, or preferably a function that explicitly checks the valid values to avoid a possible keyword flood attack.